CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.0.0-beta.3 (2021-11-16)
+
+- Fixed the StackBlitz code examples to work with Angular 12. []()
```

Comment:
```suggestion
- Fixed the StackBlitz code examples to work with Angular 12. [#182](https://github.com/blackbaud/skyux-docs-tools/pull/182)
```
I _think_ this changelog entry is referring to the changes within the same PR. Is that right?